"0","save(counts.annot.gadmor, file = ""../results/bowtie/counts.annot.gadmor"")
"
"2","Warning in gzfile(file, ""wb"") :"
"2","
 "
"2"," cannot open compressed file '../results/bowtie/counts.annot.gadmor', probable reason 'No such file or directory'
"
"2","Error in gzfile(file, ""wb"") : cannot open the connection
"
